doc/chrome-webstore-permissions-descriptions.md

```diff
@@ -0,0 +1,29 @@
+# Privacy tab in chrome web store:
+-----------------------------------
+
+## Permissions:
+
+#### Privacy -
+Privacy Badger needs access to the privacy API so that it can restrict the webRTCIPHandlingPolicy so that none of the user's local address information is exposed, as well as if they're in an incognito window, it will override and disable the alternateErrorPagesEnabled and hyperlinkAuditingEnabled settings.
+
+#### Cookies -
+Privacy Badger needs access to the cookies API in order to correct when Cloudflare domains are incorrectly identified as trackers, as well as to check for availability of the `firstPartyDomain` parameter in Firefox profiles.
```

Comment:
Can probably remove "as well as to check for availability of the `firstPartyDomain` parameter in Firefox profiles" since that's only done so that any subsequent `chrome.cookies` calls work correctly on all browsers. Cloudflare (and potentially anything else like Cloudflare where we want to do clean up based on cookie values) is the only real reason for `chrome.cookies` at this time.